### DISCUSSION TITLE
Multiple requests to the server fails

### DISCUSSION THREAD

**Discussion by 1003n40** [aquasecurity/trivy#4279] [Q&A]
When I do multiple scan jobs on kubernetes cluster (many pods which does the command `trivy image --server http://hostname:port --format json image`, some of the logs for different scans are cut, which later is a problem for me because I use the log to parse the json and if it is not consistent I can't retrieve the vulnerabilities and do my job. I thought it is problem resource wise, but setting up the server to 1Gi and 500m cpu it still fails the jobs (which act as a client command call). It happens randomly. From what I saw it is something with handling many requests which seems to cut off the request body. From also what I saw this problem doesn't occur on dockerd while on containerd runtimes it does. Should it be something with requests handling or logging dump size on different platforms.
Using Kubernetes 1.21.14.

**Comment by ngk512** [comment]
How to scan the remote images using trivy for both Linux and windows?

**Comment by 1003n40** [comment]
containerd/containerd#7289 This was reproduced in containerd, should we further investigate?
